budget/web.py

```diff
@@ -66,6 +80,23 @@ def pull_project(endpoint, values):
                     url_for(".authenticate", project_id=project_id))
 
 
+@main.route("/authenticate_admin", methods=["GET", "POST"])
```

Comment:
IMO ``admin`` would be a better fit for an administration page but since we already have a ``/dashboard`` that we could also use for administrative tasks, it makes sense to use ``admin`` here